tools/build-release-tarball

```diff
@@ -102,7 +102,12 @@ ln -nfs "$BUILD_PROD_STATIC_LOGPATH" "$USER_PROD_STATIC_LOGPATH"
 echo; echo -e "\033[33mRunning update-prod-static; check ${TMPDIR}/update-prod-static.log if it fails\033[0m"; echo
```

Comment:
probably worth cleaning up this line while you're at it, which much less helpfully says the same thing :).